docs/data/xrootd/install-storage-element.md

```diff
@@ -115,6 +115,21 @@ root@host # xrdcp /bin/sh  root://<RDRNODE>:1094///mnt/xrootd/second_test
 
 Check that the `/mnt/xrootd/second_test` is located on data server `<DATANODE>`.
 
+### (Optional) Adding High Availability (HA) redirectors
+
+It is possible to have an XRootD clustered setup with more than one redirector to ensure high availability service. To do this:
+
+1. In the `/etc/xrootd/xrootd-clustered.cfg` on each data node follow the instructions on this [section](#modify-etcxrootdxrootd-clusteredcfg) with:
+
+        :::file
+        set xrdr1 = <RDRNODE1>
+        set xrdr2 = <RDRNODE2>
+        all.manager $(xrdr1):3121
+        all.manager $(xrdr2):3121
+
+1. Create a DNS record for `<RDRNODE>` with aliases to `<RDNODE1>` and `<RDRNODE2>`
+1. The FQDN that should be advertised to users to interact with the XRootD cluster should be `<RDRNODE>`.
```

Comment:
Excuse my XRootD config ignorance but how does this step happen? if `<RDRNODE>` itself isn't mentioned int he config anywhere?

Reply:
Well since `<RDRDNODE>` is an alias that points to two IP's:

1. `<RDNODE1>`
1. `<RDRNODE2>`

The client is good smart enough to try both addresses. So in general with High Availability in XRootD the moto is: You want the infrastructure to be fully aware of the topology and from the client side you want everything to be hidden though DNS.

Reply:
```suggestion
1. Advertise the `<RDRNODE>` FQDN to users interacting with the XRootD cluster should be `<RDRNODE>`.
```

For step-by-step documentation, we should use imperative sentences